src/session.ts

```diff
@@ -666,6 +680,12 @@ export class SessionManager implements Middleware {
         this.statusBarItem.text = statusIconText + statusText;
     }
 
+    private async warnPowerShellNotFoundAndSetSessionFailure() {
```

Comment:
Is this used?

Reply:
Oh need to remove that. From my attempts to get a button into that message (not actually as easy as we thought)